### ISSUE TITLE
Github repo files were good for overview and downloading

### ISSUE THREAD

**Issue by RahulAnand442001** [appbrewery/100-days-of-python#13]
Mam It was more comfortable here as we can check resources and course goals day wise. It will be quite confusing at the Udemy website. We can see here list wise projects and match our ones too. Can you please revert it back here .

Rahul

**Comment by Rupam64** [comment]
@RahulAnand442001 You are absolutely right, Angela Mam should revert and add all the course resource files back here as it creates confusion and problems on udemy individually. If all the resources are stacked and kept in one place, then it's easy to access the files and understand the concepts in less time. I hope Angela Mam considers our point and takes the necessary actions. Thanking you.

Rupam

**Comment by zaramid** [comment]
I agree. The list of resources on Github was so comfortable to use.

**Comment by ghost** [comment]
Yes, the GitHub repo was a one-stop solution to access the resources.

**Comment by robo-monk** [comment]
https://github.com/robo-monk/100-days-of-python

**Comment by akrish4** [comment]
Yes actually i think she did this beacause those who dont buy the course but can access the code files so she wanted to make it only available to those who paid. but there is a solution for it if you have forked this repository at the early stages itself like before  this change then you would get the earlier code same as @robo-monk  has shared.
